runtime/compiler/codegen/J9RecognizedMethodsEnum.hpp

```diff
@@ -450,6 +450,7 @@
    jdk_internal_misc_Unsafe_copyMemory0,
    jdk_internal_loader_NativeLibraries_load,
    jdk_internal_util_Preconditions_checkIndex,
+   jdk_internal_util_ArraysSupport_vectorizedMismatch,
```

Comment:
minor nitpick, but check the pattern in the order for recognized method. I think this one should go before `Preconditions..`.